### PR TITLE
Changes to use mux/demux

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qp_woltka/util.py [hunk 1]
```diff
@@ -121,8 +141,12 @@ def demux(input_, output):
             if current_fp is not None:
                 current_fp.close()
 
-            current_fp = open(output + sep + fname + ext, mode)
+            sample = search_by_filename(
+                fname.decode('utf8'), lookup).encode('ascii')
+            fullname = output + sep + sample + ext
+            current_fp = open(fullname, mode)
             current_fname = fname
+            print(fullname.decode('ascii'))
```
Comment (wasade) [inline qp_woltka/util.py:149]: Is this print needed?

File: qp_woltka/woltka.py [hunk 2]
```diff
@@ -282,29 +154,92 @@ def woltka_to_array(directory, output, database_bowtie2,
              # includes ignoring files that have "0.00% overall alignment rate"
              # in their log file, which is when nothing aligns and is fine to
              # skip
-             "PROCESS=1; COUNTER=0; for f in `awk '{print $NF}' "
-             f'{output}/*.array-details`; do let COUNTER=COUNTER+1; '
-             "if [ ! -f ${f}*/free.biom ]; then if ! grep -xq "
-             "'0.00% overall alignment rate' *_${COUNTER}.err; "
-             "then PROCESS=0; fi; fi; done",
-             "if [ 1 -eq $PROCESS ]; then ",
+             # # "PROCESS=1; COUNTER=0; for f in `awk '{print $NF}' "
```
Comment (wasade) [inline qp_woltka/woltka.py:157]: Should these comments be deleted?

**Review by wasade — COMMENTED** [review]
Just minor questions, otherwise looks good!

File: qp_woltka/util.py (hunk 1 above)
Comment (antgonza) [reply qp_woltka/util.py:149]: Yes, that's how we keep track of the samples/files that were processed.

File: qp_woltka/woltka.py (hunk 2 above)
Comment (antgonza) [reply qp_woltka/woltka.py:157]: No, we don't need them but those are the lines that are actually checking for failed samples so I would like to keep them until we have something in place to replace them.

**Review by antgonza — COMMENTED** [review]
@wasade, thank you for your review.

**Comment by antgonza** [comment]
With all @wasade's blessings, I'm merging! Thank you!

**Pull request merged**